### ISSUE TITLE
Tutorial1_OMIB.ipynb

### ISSUE THREAD

**Issue by fnbillimoria** [Energy-MAC/LITS-Examples#6]
Dear Rodrigo / Jose,

I am researcher from Oxford attempting to run the Tutorial1_OMIB.ipynb test case on LITS as per the instructions.    On the initial definition of the bus vectors, I receive a conversion error "ERROR: MethodError: Cannot `convert` an object of type String to an object of type PowerSystems.BusTypes.BusType".  Would you have any clues as to what the type mismatch may be related to?    I have set out the full error message below.

Many thanks,
Farhad

      ];
ERROR: MethodError: Cannot `convert` an object of type String to an object of type PowerSystems.BusTypes.BusType
Closest candidates are:
  convert(::Type{PowerSystems.BusTypes.BusType}, ::PowerSystems.MatpowerBusTypes.MatpowerBusType) at C:\Users\fbillimoria\.julia\packages\PowerSystems\wHSk9\src\parsers\power_models_data.jl:98
  convert(::Type{S}, ::T) where {S, T<:(Union{CategoricalArrays.CategoricalString{R}, CategoricalArrays.CategoricalValue{T,R} where T} where R)} at C:\Users\fbillimoria\.julia\packages\CategoricalArrays\dmrjI\src\value.jl:103
  convert(::Type{T}, ::T) where T at essentials.jl:168
Stacktrace:
 [1] convert(::Type{Union{Nothing, PowerSystems.BusTypes.BusType}}, ::String) at .\some.jl:34
 [2] Bus at C:\Users\fbillimoria\.julia\packages\PowerSystems\wHSk9\src\models\generated\Bus.jl:71 [inlined]
 [3] Bus at C:\Users\fbillimoria\.julia\packages\PowerSystems\wHSk9\src\models\generated\Bus.jl:76 [inlined] (repeats 2 times)
 [4] top-level scope at none:0

**Comment by jd-lara** [comment]
Hi @fnbillimoria sorry, we haven't updated the tutorials. Recent updates to the data structs in PowerSystems.jl made some of the data definitions obsolete. We will update and get back to these issues asap.

**Comment by jd-lara** [comment]
close via 386f5833e975f324e72390f0120d6f96bbd537e6

**Comment by jd-lara** [comment]
@fnbillimoria we have updated the examples now. Please don't hesitate to open the issue again if you still have problems

**Comment by fnbillimoria** [comment]
Dear Jose - Thank you for your quick attention to this.  The first issue has been resolved, but now getting an error on solving the powerflow.  Looks to be a method matching error relating to the PowerSystems load flow.  Not sure if I am running correctly but any advice greatly appreciated.  Full error replicated below.   Many thanks - Farhad

LoadError: MethodError: no method matching _ybus!(::SparseArrays.SparseMatrixCSC{Complex{Float64},Int32}, ::Line, ::Dict{Int64,Int64})
Closest candidates are:
  _ybus!(!Matched::SparseArrays.SparseMatrixCSC{Complex{Float64},Int64}, ::Line, ::Dict{Int64,Int64}) at C:\Users\fbillimoria\.julia\packages\PowerSystems\y1jmI\src\utils\network_calculations\ybus_calculations.jl:13
  _ybus!(!Matched::SparseArrays.SparseMatrixCSC{Complex{Float64},Int64}, !Matched::FixedAdmittance, ::Dict{Int64,Int64}) at C:\Users\fbillimoria\.julia\packages\PowerSystems\y1jmI\src\utils\network_calculations\ybus_calculations.jl:116
  _ybus!(!Matched::SparseArrays.SparseMatrixCSC{Complex{Float64},Int64}, !Matched::PhaseShiftingTransformer, ::Dict{Int64,Int64}) at C:\Users\fbillimoria\.julia\packages\PowerSystems\y1jmI\src\utils\network_calculations\ybus_calculations.jl:88
  ...
in expression starting at C:\Users\fbillimoria\.atom\LITS_example_1.jl:24
_buildybus(::InfrastructureSystems.FlattenIteratorWrapper{ACBranch}, ::Array{Bus,1}, ::Array{FixedAdmittance,1}) at ybus_calculations.jl:141
Ybus(::InfrastructureSystems.FlattenIteratorWrapper{ACBranch}, ::Array{Bus,1}) at ybus_calculations.jl:160
make_pf(::System) at make_pf.jl:41
#solve_powerflow!#124(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::typeof(solve_powerflow!), ::System, ::Module) at power_flow.jl:143
solve_powerflow!(::System, ::Module) at power_flow.jl:143
top-level scope at LITS_example_1.jl:24

**Comment by jd-lara** [comment]
Which version of PowerSystems are you currently running

**Comment by jd-lara** [comment]
@fnbillimoria please find the running binder here 

https://hub.gke.mybinder.org/user/energy-mac-lits-examples-kwxrqmjt/notebooks/Tutorial1_OMIB.ipynb

I re-ran the code and it is working

**Comment by fnbillimoria** [comment]
Hi Jose - I just tried the link and the link appears broken.   I just updated my PowerSystems install, and tried again with the same error.  It looks to be a Method matching error - with the ybus function in PowerSystems - seems to be expecting an argument that is Int64 but instead is getting Int32 (see below).   Perhaps it is to do with whether I have downloaded the 32 or 64 bit version of Julia.
  
(::SparseArrays.SparseMatrixCSC{Complex{Float64},Int32}
versus
SparseArrays.SparseMatrixCSC{Complex{Float64},Int64},
